docs/faq/faq.md

```diff
@@ -418,6 +421,14 @@ The Accelerator provided sample security groups in the workload accounts offer a
 
 The use of NACLs are general discouraged, but leveraged in this architecture as a defense-in-depth mechanism.  Security groups should be used as the primary access control mechanism.  As with security groups, we encourage customers to review and tailor their NACLs based on their own security requirements.
 
+### 1.6.8. Can I deploy the solution as root?
+
+No you cannot install as root as root has no ability to assume roles. This would prevent the deployment.
+
+### 1.6.9. Can GuardDuty monitor the Organizational Management root acccount from the Security account?
+
+Yes - enabled by default.
```

Comment:
Please elaborate.  what specifically is enabled by default.  How is this possible. ...